setup/artifacts/gcs.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: single
+title:  "Google Cloud Storage Artifacts"
+sidebar:
+  nav: setup
+---
+
+{% include toc %}
+
+Spinnaker stages that support reading data from artifacts can consume [Google Cloud
+Storage](https://cloud.google.com/storage/) (GCS) objects as artifacts.
```

Comment:
Suggest just GCS, instead of spelling out. It's already spelled out in the page title, and I'm pretty sure your audience is gonna get it. And the sentence is a little unwieldy with "can consume" coming right after artifacts, even though it's the stages that can consume. 

Maybe instead of "...stages that support reading..." say "...stages that read..."? That puts the subject closer to the main verb--stages can consume.